ops/testing.py

```diff
@@ -1890,6 +1890,42 @@ def run_action(self, action_name: str,
                 output=action_under_test.output)
         return action_under_test.output
 
+    def set_cloud_spec(self, content: Dict[str, str]):
+        """Set cloud specification (metadata) including credentials.
+
+        Call this method before trying to call `Harness.Model.get_cloud_spec`.
+
+        Example usage::
+
+            class TestCharm(unittest.TestCase):
```

Comment:
We're encouraging charm tests to be written with pytest, so we should use that in examples now.

Reply:
Hmm, this will be inconsistent with the rest of the docs, though. Maybe have this be unittest to match the rest, and we open a ticket (or add to the existing one) changing the API docs? It seems like a fairly small job and I would rather see them all be using the same style.

Reply:
@benhoyt should we create a GitHub issue/Jira ticket for this?

Reply:
Yep, that sounds good, thanks @IronCore864. Maybe you can assign that new issue to yourself to do later. Should be a fairly quick fix to our examples.

Reply:
https://github.com/canonical/operator/issues/1154